app/assets/javascripts/application/slides.js

```diff
@@ -1,12 +1,17 @@
 Utils.fireWhenReady(['slides#preview'], function(e) {
   // Scale the view to the size it will need to display
-  $('.ui-feed').css({width: '1920px', height: '1080px'});
+  var $uiFeed = $('.ui-feed');
+  var isVertical = $uiFeed.find(".js-slide-meta").data("orientation") == "vertical" ? true : false;
+  var stdWidth = isVertical? 1080 : 1920;
+  var stdHeight = isVertical? 1920 : 1080;
```

Comment:
Note source of magic values here and elsewhere. I'd prefer they were their own vars. Something like:

    // Product owners provided these values. They are our standard monitors dimensions.
    var STD_LONG_DIM = 1920;
    var STD_SHORT_DIM = 1080;